src/lib.rs

```diff
@@ -56,8 +56,8 @@ use syntax::preprocess::*;
 use syntax::ast::*;
 use data::input_stream::InputStream;
 use data::position::Position;
-use parser::parser_monad::ParseError;
-use parser::parser::parseC;
+use parser::parser_utils::ParseError;
```

Comment:
`parser_utils.rs` ultimately looks like it could just be `mod.rs`, if we wanted to avoid the less descript "utils" identifier.